freqtrade/strategy/interface.py

```diff
@@ -476,8 +481,22 @@ def get_signal(self, pair: str, timeframe: str, dataframe: DataFrame) -> Tuple[b
         (buy, sell) = latest[SignalType.BUY.value] == 1, latest[SignalType.SELL.value] == 1
         logger.debug('trigger: %s (pair=%s) buy=%s sell=%s',
                      latest['date'], pair, str(buy), str(sell))
+        timeframe_seconds = timeframe_to_seconds(timeframe)
+        if self.ignore_expired_candle(latest_date=latest_date,
+                                      timeframe_seconds=timeframe_seconds,
+                                      buy=buy):
+            return False, sell
         return buy, sell
 
+    def ignore_expired_candle(self, latest_date: datetime, timeframe_seconds: int, buy: bool):
```

Comment:
Calculation seems correct now - at least doing the math manually...

I'll probably add a test-case testing this from another angle when i get time to "play" with this ...